src/systems/ec/implementations/react/components/select/src/Select.jsx

```diff
@@ -127,6 +134,7 @@ Select.propTypes = {
     PropTypes.shape({
       value: PropTypes.string,
       label: PropTypes.string,
+      attributes: PropTypes.array,
```

Comment:
```suggestion
      isDisabled: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
```
Basing on the previously added `disabled={option.isDisabled}`

Reply:
Not needed.